system/modules/isotope/dca/tl_iso_config.php

```diff
@@ -350,7 +350,8 @@
                 (
                     'name' => array
                     (
-                        'label'                 => ['&nbsp;'],
+                        'label'                 => ['&nbsp;', ''],
+                        'inputType'             => 'text',
```

Comment:
Yes, that's correct.
I was wondering if I should open a PR there or adjust the code here so that the error doesn't occur.
I have now reversed the change.